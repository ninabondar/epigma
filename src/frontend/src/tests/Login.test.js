import React from 'react';
import Login from '../components/Login'

describe('<Login/>', () => {
  test('should match its snapshot', () => {
    const component = shallow(<Login/>)
    expect(component).toMatchSnapshot()
  })
})