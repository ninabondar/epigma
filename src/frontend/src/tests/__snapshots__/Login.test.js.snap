// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> should match its snapshot 1`] = `
<form
  className="Login"
>
  <input
    id="Login__username"
    placeholder="login"
    required={true}
    type="text"
  />
  <input
    id="Login__password"
    placeholder="password"
    required={true}
    type="password"
  />
  <button
    className="Login__submit-btn"
  >
    Proceed
  </button>
</form>
`;
